src/client/wallet/PowerUpTransaction.js

```diff
@@ -1,16 +1,23 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
 import { FormattedMessage, FormattedRelative, FormattedDate, FormattedTime } from 'react-intl';
 import BTooltip from '../components/BTooltip';
 
-const PowerUpTransaction = ({ timestamp, amount }) => (
+const PowerUpTransaction = ({ timestamp, amount, to, from }) => (
   <div className="UserWalletTransactions__transaction">
     <div className="UserWalletTransactions__icon-container">
       <i className="iconfont icon-flashlight_fill UserWalletTransactions__icon" />
     </div>
     <div className="UserWalletTransactions__content">
       <div className="UserWalletTransactions__content-recipient">
         <FormattedMessage id="powered_up" defaultMessage="Powered up " />
+        {to === from ? null : (
```

Comment:
We can use `to !== from &&` instead of ternary.

Reply:
Yes, that's better. Squashed the commit with your suggestions. Thank you. :)